hack/def/kind.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Jetstack cert-manager contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KIND="$2"
+NAME="$1"
+IMAGE="$3"
+CONFIG="$4"
+OUTPUT="$5"
+
+# We set PATH to contain /usr/local/bin in order to access the 'docker' CLI
+# binary when running 'kind cluster create' within Bazel.
+# If different OS store the docker binary at a different path, this line will
+# need to be updated.
+export PATH="$PATH:/usr/local/bin"
+
+KIND_KUBECONFIG=$("$KIND" get kubeconfig-path --name $NAME)
+PERSISTENT_TMP="/tmp/certmanager/kind-$NAME"
```

Comment:
This is one not-so-great part of this whole script 🙈

Reply:
We will be able to remove this once/if kind comes up with a way to run `kind get kubeconfig` to generate a *new* kubeconfig file for an existing cluster (cc @BenTheElder)